tests/by-util/test_basename.rs

```diff
@@ -84,3 +85,23 @@ fn test_no_args() {
 fn test_too_many_args() {
     expect_error(vec!["a", "b", "c"]);
 }
+
+fn test_invalid_utf8_args(os_str: &OsStr) {
```

Comment:
Missing #[test] ?

Reply:
called by line 242: 
`#[cfg(any(unix, target_os = "redox"))]
    #[test]
    fn invalid_utf8_args_unix() {
        use std::os::unix::ffi::OsStrExt;

        let source = [0x66, 0x6f, 0x80, 0x6f];
        let os_str = OsStr::from_bytes(&source[..]);
        test_invalid_utf8_args_lossy(os_str);
        test_invalid_utf8_args_ignore(os_str);
    }`
Allows for no code duplication with illegal windows wide-string encoding (not implemented yet due to missing windows machine)

Reply:
@sylvestre